editions/tw5.com/tiddlers/wikitext/Formatting in Wikitext Semantics.tid

```diff
@@ -0,0 +1,37 @@
+created: 20230303205650613
+modified: 20230303211541555
+tags: [[Formatting in WikiText]]
+title: Formatting in Wikitext Semantics
```

Comment:
I don't think splitting off a separate tiddler about semantics works here. It makes it harder to understand overall because one needs to jump between the two tiddlers to understand.

As I said above, I think the HTML spec stuff about strong/emphasis should just be addressed via brief note.